string robot_name
int64 task_index
---
int8[] node_list